src/__test__/index.test.ts

```diff
@@ -60,8 +57,8 @@ const createPerson = (): Person => {
 }
 
 describe('Client', function() {
-  const bytes = Buffer.concat([Buffer.from(encode(0x01)), Buffer.from(encode(0x55)), randomBytes(32)])
-  const dbID = multibase.encode('base32', bytes).toString()
+  const threadId = ThreadID.fromRandom()
+  const dbID = threadId.stringOfBase('base32')
```

Comment:
Ah I see. The method you wanted here was `string()` (which does default to base32). But I've created an issue to actually switch to `toString()` for all this to simplify things.

Reply:
ah okay. same outcome. i'll change it with future update